src/VisualStudio/Core/Next/Remote/JsonRpcSession.cs

```diff
@@ -16,87 +16,35 @@ namespace Microsoft.VisualStudio.LanguageServices.Remote
 {
     internal class JsonRpcSession : RemoteHostClient.Session
```

Comment:
can you explain why we need sessions and JsonRpc sessions?  Could it not be simpler to just have one session concept that people can talk over (using json, or some direct streaming approach, depending on their preference)?   All this subclassing is making my head hurt.